src/ai/ai_info.hpp

```diff
@@ -12,6 +12,9 @@
 
 #include "../script/script_info.hpp"
 
+/* All valid AI versions, in order. */
+const std::list<std::string> AI_VERSIONS = { "0.7", "1.0", "1.1", "1.2", "1.3", "1.4", "1.5", "1.6", "1.7", "1.8", "1.9", "1.10", "1.11", "12", "13", "14", "15" };
```

Comment:
```suggestion
static constexpr std::initializer_list<std::string> AI_VERSIONS{ "0.7", "1.0", "1.1", "1.2", "1.3", "1.4", "1.5", "1.6", "1.7", "1.8", "1.9", "1.10", "1.11", "12", "13", "14", "15" };
```
Please don't use list as it's a linked list. I've used `std::initializer_list` as that doesn't copy the data table.

Reply:
Oops; a `std::vector` would even have been better than `std::list` I guess. My C++ is rustyyyy.

PS: also has to be a `string_view`, but tomato tomato! Tnx :)